qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -0,0 +1,1042 @@
+
+        Returns:
+            Experiment errors.
+        """
+        self.status()  # Collect new errors.
+        return "\n".join(self._errors)
+
+    def tags(self) -> List[str]:
+        """Return tags assigned to this experiment data.
+
+        Returns:
+            A list of tags assigned to this experiment data.
+
+        """
+        return self._tags
+
+    @auto_save
+    def set_tags(self, new_tags: List[str]) -> None:
+        """Set tags for this experiment.
+
+        Args:
+            new_tags: New tags for the experiment.
+        """
+        self._tags = new_tags
+
+    def metadata(self) -> Dict:
+        """Return experiment metadata.
+
+        Returns:
+            Experiment metadata.
+        """
+        return self._metadata
+
+    @auto_save
+    def set_metadata(self, metadata: Dict) -> None:
+        """Set metadata for this experiment.
+
+        Args:
+            metadata: New metadata for the experiment.
+        """
+        self._metadata = copy.deepcopy(metadata)
+
+    @property
+    def _provider(self) -> Optional[Provider]:
+        """Return the provider.
+
+        Returns:
+            Provider used for the experiment, or ``None`` if unknown.
+        """
+        if self._backend is None:
+            return None
+        return self._backend.provider()
+
+    @property
+    def experiment_id(self) -> str:
+        """Return experiment ID
+
+        Returns:
+            Experiment ID.
+        """
+        return self._id
+
+    @property
+    def job_ids(self) -> List[str]:
+        """Return experiment job IDs.
+
+        Returns: IDs of jobs submitted for this experiment.
+        """
+        return self._jobs.keys()
+
+    @property
+    def backend(self) -> Optional[Union[BaseBackend, Backend]]:
+        """Return backend.
+
+        Returns:
+            Backend this experiment is for, or ``None`` if backend is unknown.
+        """
+        return self._backend
+
+    @property
+    def experiment_type(self) -> str:
+        """Return experiment type.
+
+        Returns:
+            Experiment type.
+        """
+        return self._type
+
+    @property
+    def figure_names(self) -> List[str]:
+        """Return names of the figures associated with this experiment.
+
+        Returns:
+            Names of figures associated with this experiment.
+        """
+        return self._figures.keys()
+
+    @property
+    def share_level(self) -> str:
+        """Return the share level fo this experiment.
+
+        Returns:
+            Experiment share level.
+        """
+        return self._share_level
+
+    @share_level.setter
+    def share_level(self, new_level: str) -> None:
+        """Set the experiment share level.
+
+        Args:
+            new_level: New experiment share level. Valid share levels are provider-
+                specified. For example, IBM Quantum experiment service allows
+                "global", "hub", "group", "project", and "private".
```

Comment:
```suggestion
                "public", "hub", "group", "project", and "private".
```

Reply:
It was `global` in `ibmq-provider` to match circuit job share levels... but that has since gone away so I guess we can go back to `public` lol. Fixed in 2e01e94